### PR TITLE
Fix URL for libcomm14cux

### PR DESCRIPTION
**Pull request by colinbourassa**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1550,7 +1550,7 @@ <h2 id="packages">List of Packages</h2>
     </tr>
     <tr>
         <td class="package">libcomm14cux</td>
-        <td class="website"><a href="https://www.github.org/colinbourassa/libcomm14cux/">libcomm14cux</a></td>
+        <td class="website"><a href="http://github.org/colinbourassa/libcomm14cux/">libcomm14cux</a></td>
```
Comment (TimothyGu) [inline index.html]: github.org?

**Comment by colinbourassa** [comment]
Sorry about that. I've corrected it to github.com in the second commit on this branch.

**Comment by tonytheodore** [comment]
Thanks, could you squash this into a single commit?

**Comment by colinbourassa** [comment]
All set.

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**